recipe/meta.yaml

```diff
@@ -32,14 +32,22 @@ test:
     - pip check
   requires:
     - pip
-    - python {{ python_min }}
```

Comment:
Can we run some tests https://github.com/lchenn/py-grpc-prometheus/tree/v0.3.0/tests?